src/libappimage/desktop_integration/integrator/Integrator.cpp

```diff
@@ -79,7 +78,7 @@ namespace appimage {
                         }
                     } catch (const XdgUtils::DesktopEntry::BadCast& err) {
                         // if the value is not a bool we can ignore it
-                        logger.warning() << err.what() << std::endl;
+                        utils::Logger::warning(err.what());
```

Comment:
You already use the namespace, no need for the prefix.

Reply:
fixed